pyproject.toml

```diff
@@ -26,7 +27,6 @@ pytest-cov = "^5.0.0"
 pytest-mock = "^3.14.0"
 pytest-env = "^1.1.3"
 pytest-asyncio = "^0.23.6"
-httpx = "^0.24.1"
```

Comment:
oh yeah, this definitely doesn't belong in this group, although I think we need to add it to the core group since one of the lei checking functions needs it

Reply:
It comes along with fastapi/starlette so the package is there, even though we're not explicitly defining it as a depdendency